api/models/event/mixin.js

```diff
@@ -66,7 +66,7 @@ export default {
       }
     }
 
-    return from + to
+    return from + to + " UTC timezone"
```

Comment:
is the emails the only place this is used, just want to make sure this isnt weird somewhere else? also i dont think we need "timezone", UTC is fine

Reply:
Ok. And yes, only used in the context of emails.